docs/sources/tempo/traceql/query-editor.md

```diff
@@ -11,20 +11,66 @@ keywords:
   - TraceQL
 ---
 
-# TraceQL query editor
+# Create TraceQL queries in Grafana
```

Comment:
```suggestion
# Write TraceQL queries in Grafana
```

"Create" sounds to me like something intended to last when here it's about exploring data, something not necessarily intended to be persisted.

Reply:
Thank you for the clarification.